docs/kratos/guides/tracing.mdx

```diff
@@ -9,15 +9,17 @@ It's yet another tool that you can use to aid you in profiling, debugging and
 ultimately understanding your deployment of Kratos better. Kratos supports the
 following tracing options:
 
-- Tracing backend(s): Jaeger, Elastic APM, Datadog, Zipkin and Instana - _Note:
```

Comment:
Are all of these no longer supported? If not, is there an issue tracking this and can we link it here?

Reply:
Official support in OpenTelemetry exists only for the backends listed here: https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters. Other backends can be technically "supported" by sending spans to the OpenTelemetry Collector agent, and configuring the collector to push spans to whatever backend of your choice (see: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter for a list of supported third-party backends).

Also, some providers (like Elastic APM, idk any other off the top of my head) are now supporting the OpenTelemetry Protocol (OTLP) directly, so you can simply use the [otlp/otlptrace](https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters/otlp/otlptrace) exporter and point it to your Elastic APM endpoint.

Reply:
Yes, that would be perfect! We then also need to add it to the release notes. To do this, please create an empty commit with the message in Ory Kratos in the format of:

```
refactor: <some title>

<description>

BREAKING CHANGE: To upgrade your Zipkin tracer do ...
```

with `git commit -a --allow-empty`

and make a PR against ory/kratos so that we can include the content